runtime/doc/version9.txt

```diff
@@ -43,6 +43,17 @@ when the release was being prepared.  Sven was a long time supporter of Vim.
 He registered the vim.org domain and created the first Vim website.  We will
 remember him!
 
+							*Bram-Moolenaar* *Bram*
+Vim version 9.1 is dedicated to Bram Moolenaar, who passed away August 3rd 2023
```

Comment:
"who passed away on"?

Reply:
I would suggest "who passed away August 3rd, 2023". The "on" is optional to my (non-native) eyes.

Reply:
You mean adding "," between the date and the year?

Reply:
@k-takata he means remove the word "on"

Reply:
English is not my native language. I suggest.

> Vim version 9.1 is dedicated to Bram Moolenaar, who passed away `on` August 3rd 2023